src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (item1, item2) => {
```

Comment:
default compare function should also return 0 if values are equal to each other
also, let's define default function higher, instead of doing it in arguments